src/uct/sm/mm/base/mm_ep.c

```diff
@@ -320,7 +342,7 @@ static UCS_F_ALWAYS_INLINE ssize_t uct_mm_ep_am_common_send(
     switch (send_op) {
     case UCT_MM_SEND_AM_SHORT:
         /* write to the remote FIFO */
-        uct_am_short_fill_data(elem + 1, header, payload, length);
+        uct_mm_ep_short_fill_data(elem + 1, header, payload, length);
```

Comment:
Initially, I thought not to modify uct_am_short_fill_data() because other UCTs used it.
Adding a hint param should be fine, because with hint as '0' it will fallback to memcpy().